README.md

```diff
@@ -26,60 +27,60 @@ They are used to achieve maximum security, privacy and comfortable use of the In
 * Can block "adult" sites *
 * Hides visited sites from your provider
 * Prevents some types of resource blocks
-* Open-source
+* Libre software
```

Comment:
Does everyone really know what libre software is?

Reply:
No, but it is all there in the word (as opposed to "open source".